pkg/kubectl/cmd/cmd_test.go

```diff
@@ -201,11 +201,17 @@ func NewAPIFactory() (*cmdutil.Factory, *testFactory, runtime.Codec) {
 		Validator: validation.NullSchema{},
 	}
 	generators := map[string]kubectl.Generator{
```

Comment:
sure, right now `Factory` does not expose its internal map, so I would need to add something to at minimum let you enumerate its internal generator names